cvmfs/cvmfs_config

```diff
@@ -1466,6 +1473,10 @@ cvmfs_bugreport() {
   mkdir etc
   cp -r /etc/cvmfs etc/
 
+  echo "Gathering /var/run/cvmfs"
+  mkdir -p var/run/cvmfs
+  cp -r /var/run/cvmfs var/run/cvmfs
```

Comment:
```
tar: bugreport/var/run/cvmfs/cvmfs/cvmfs.cvmfs-config.cern.ch: socket ignored
```
Let's silence this  - maybe with a `find -type f -exec cp ...`

Reply:
sadly not a single line replacement to just surpress the warning